nuitka/plugins/standard/PlaywrightPlugin.py

```diff
@@ -41,6 +42,15 @@ def addPluginCommandLineOptions(cls, group):
             Playwright browser to include. Can be specified multiple times. use "all" to include all installed browsers.
             """,
         )
+        group.add_option(
+            "--playwright-exclude-browser",
```

Comment:
I think we agreed that the want to just have a value `none` for the included browsers, which will then give no error exit, and resolve to empty data files, etc.